libs/core/src/lib/card/kpi/card-kpi-value.directive.ts

```diff
@@ -12,6 +14,10 @@ export class CardKpiValueDirective implements OnInit, CssClassBuilder {
     /** @hidden */
     class: string;
 
+    /** Set type of KPI value. eg: 'positive', 'negative', 'critical', 'informative' */
+    @Input()
+    fdType: KpiValueType;
```

Comment:
`fdType` is not a good name for this property. Something like `kpiType` or `kpiStatus` or just `status` would be more self-explanatory.

Reply:
renamed property to status.